Replace 'attribute' with 'property' for 'indeterminate' (#31737)
mdn/content: files/en-us/web/css/_colon_indeterminate/index.md

{{CSSRef}}

> The **`:indeterminate`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) represents any form element whose state is indeterminate, such as checkboxes which have their [`indeterminate`](/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes) JavaScript property set to `true`, radio buttons which are members of a group in which all radio buttons are unchecked, and indeterminate {{HTMLElement("progress")}} elements.

Reviewer: ```suggestion
The **`:indeterminate`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) represents any form element whose state is indeterminate, such as checkboxes that have been set to an [`indeterminate`](/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes) state with JavaScript, radio buttons which are members of a group in which all radio buttons are unchecked, and {{HTMLElement("progress")}} elements with no `value` attribute.
```
While your statement was factual, the indeterminate IDL page doesn't exist (see the non-link on https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement) so we can't link to it so explained how we get to `true` instead of using `true` with no link. Also explained what an indeterminate progress is, instead of using the term people may not understand.

Author: Fair points, I applied the suggestion.
